matrix/common/etl/transformers/cell_expression.py

```diff
@@ -20,6 +20,10 @@
 class CellExpressionTransformer(MetadataToPsvTransformer):
     """Reads SS2 and 10X bundles and writes out rows for expression and cell tables in PSV format."""
 
+    # The minimum UMI count in the emptydrops result required to include a
+    # putative cell in the matrix service.
+    emptydrops_min_count = 100
```

Comment:
Would be great to allow the users to adjust this parameter from the API!

Reply:
Hey @nh3, so the user can adjust this via the API through the `total_umis` filter. They just couldn't set that value lower than 100. Thus far, most of the feedback that we've gotten is that 100 UMIs is a safe lower bound for users who want to query the matrix service for _cells_, which is the main use case we're trying to solve. Users who want the full set of droplets usually want to go back to raw data anyway, so wouldn't really be using matrix service outputs.